wikimedia/wikipedia-ios
# remove outer workspace file

**Fixes Phabricator ticket:** N/A

### Notes
* From engineering sync meeting - to clarify which file you sould be opening the project in

### Test Steps
1. Open project.
2. Run app, ensure it compiles/works as normal
